cuda/test/matrix/bccoo_kernels.cpp

```diff
@@ -0,0 +1,406 @@
+/*******************************<GINKGO LICENSE>******************************
```

Comment:
Since this file is not used at all, I would suggest to completely remove it for now and add it again after you have a CUDA implementation.

Reply:
I have commented the line in cuda/test/matrix/CMakeLists.txt